commands/system.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+var SystemCmd = &cobra.Command{
+	Use:   "system",
+	Short: "System management",
+}
+
+var SystemGetBusyCmd = &cobra.Command{
+	Use:     "getbusy",
+	Short:   "Get the current busy state",
+	Long:    `Gets the server busy state (high load) and timestamp corresponding to when the server busy flag will be automatically cleared.`,
+	Example: `  system getbusy`,
+	RunE:    withClient(getBusyCmdF),
+}
+
+var SystemSetBusyCmd = &cobra.Command{
+	Use:     "setbusy [seconds]",
+	Short:   "Set the busy state to true",
+	Long:    `Set the busy state to true for the specified number of seconds, which disables non-critical services.`,
+	Example: `  system setbusy 3600`,
+	Args:    cobra.ExactArgs(1),
+	RunE:    withClient(setBusyCmdF),
+}
+
+var SystemClearBusyCmd = &cobra.Command{
+	Use:     "clearbusy",
+	Short:   "Clears the busy state",
+	Long:    `Clear the busy state, which re-enables non-critical services.`,
+	Example: `  system clearbusy`,
+	RunE:    withClient(clearBusyCmdF),
+}
+
+func init() {
+	SystemSetBusyCmd.Flags().Uint("seconds", 3600, "Number of seconds until server is automatically marked as not busy.")
+	SystemCmd.AddCommand(
+		SystemGetBusyCmd,
+		SystemSetBusyCmd,
+		SystemClearBusyCmd,
+	)
+	RootCmd.AddCommand(SystemCmd)
+}
+
+func getBusyCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	printer.SetSingle(true)
+
+	sbs, response := c.GetServerBusy()
+	if response.Error != nil {
+		printer.PrintError("Unable to get busy state: " + response.Error.Error())
+		return response.Error
+	}
+	printer.PrintT("busy:{{.Busy}} expires:{{.Expires}}", sbs)
+	return nil
+}
+
+func setBusyCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		args = []string{"3600"}
+	}
+	seconds, err := strconv.ParseUint(args[0], 10, 32)
+	if err != nil || seconds == 0 {
+		err = fmt.Errorf("seconds must be a number > 0")
+		printer.PrintError(err.Error())
+		return err
+	}
+
+	ok, response := c.SetServerBusy(int(seconds))
+	if response.Error != nil || !ok {
+		printer.PrintError(fmt.Sprintf("Unable to set busy state: %v", response.Error))
+		return response.Error
+	}
+	printer.Print("Busy state set")
+	return nil
+}
+
+func clearBusyCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	ok, response := c.ClearServerBusy()
+	if response.Error != nil || !ok {
+		printer.PrintError(fmt.Sprintf("Unable to clear busy state: %v", response.Error))
```

Comment:
There is the case of `response.Error == nil && ok == false` in which case we would be printing `nil` as part of the error message

Reply:
Ugh, right.  I don't remember why I made that API return a bool and an error.  Seems redundant now.

Reply:
Fixed via https://github.com/mattermost/mmctl/pull/130/commits/67d1566f460ead013fae90c82c7b37bb3e9f18a0    I double checked that the bool mirrors the error exactly (never returns false without error) so I'm just ignoring the bool.  Changing that API would be breaking so the behaviour won't change.  I don't know what I was thinking having that return a bool in the first place.

Reply:
I think I've seen the pattern someplace else in our API. I agree with ignoring the error being a good solution